contents/handbook/product/ai-playbook.md

```diff
@@ -0,0 +1,71 @@
+---
+title: In-product AI playbook
+sidebar: Handbook
+showTitle: true
+---
+
+Throughout the 2020s, LLM-enabled products will eat those which stay in the past – just see Cursor vs. JetBrains. LLM-based PostHog features already are advancing [our mission](/handbook/why-does-posthog-exist), but this journey has just begun. New use cases are becoming viable as models improve in reliability and cost, and YC _will_ fund every AI-oriented founder with the ambition to overtake us. We can't be any less ambitious.
+
+This doc describes how PostHog goes on to thrive, instead of being eaten. You're not looking at a high-level exec strategy. You're looking at the **in-product AI playbook** for every team at PostHog.
+
+There are two angles to our in-product AI:
+
+## 1. PostHog Max – the world's first artificial product manager
+
+Product-minded developers want to build the right thing. They already can _build_, and AI dev tooling – like Cursor's agent – helps build faster.
+
+PostHog's agent helps build _the right thing_. Named Max, this is our core AI product. It's [deep research](https://openai.com/index/introducing-deep-research/) for product questions, with knowledge of data and tools living in PostHog.
+
+Max is owned by the [AI Product Manager team](/teams/ai-product-manager) – see its page for this product's roadmap.
+
+## 2. PostHog Intelligence - acceleration throughout
```

Comment:
@corywatilo What do you think of this branding of PostHog Max vs. PostHog Intelligence?

Still trying to figure out how to reconcile our out-of-product AI (Max in community questions / Chat with Max AI from the homepage) with in-app. That's a _bunch_ of different Maxes, and when someone says "Max" – we never know if they're talking about the website thing or the app thing. I don't think this is good for users either

PROPOSAL: We rename the website Max to **Min AI**, and app Max stays **Max AI** – I'm LOLing, but I'm also dead serious

Reply:
I _love_ two hedgehogs named `Min` and `Max`...later we can have `Mode` and `Mean` lol maybe `Mean` is some troll

Reply:
The idea's _on hold_, post-#10679